test/index.ts

```diff
@@ -890,7 +941,6 @@ if (!popsicle.browser) {
     t.test('should follow 303 redirect with post', function (t) {
       return popsicle.post({
         url: REMOTE_URL + '/redirect/code/303',
-        body: { foo: 'bar' }
```

Comment:
Had to remove this line for the test to work. With the line present, the test hangs, and I have no idea why. Any ideas?